hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockOutputStream.java

```diff
@@ -595,7 +595,7 @@ private void setIoException(Exception e) {
       IOException exception =  new IOException(EXCEPTION_MSG + e.toString(), e);
       ioException.compareAndSet(null, exception);
     } else {
-      LOG.debug("Previous request had already failed with " + ioe.toString()
+      LOG.debug("Previous request had already failed with " + ioe
```

Comment:
please use parameterized logging.
For example, LOG.debug("Previous request had already failed with {}", ioe)